src/app/modules/gb-analysis-module/fractalis-components/gb-fractalis-control/gb-fractalis-control.component.ts

```diff
@@ -10,26 +12,92 @@ import {ChartType} from '../../../../models/chart-models/chart-type';
 })
 export class GbFractalisControlComponent implements OnInit {
 
-  public availableChartTypes: SelectItem[];
-  public selectedChartType: string;
+  dragCounter = 0;
 
   // TODO: accept drag & drop of variables
```

Comment:
Looks like it's done, right? this comment can be removed?

Reply:
indeed, will remove